docs/UserGuide.adoc

```diff
@@ -146,20 +206,40 @@ Selects the task identified by the index number used in the last task listing. +
 Format: `select INDEX`
 
 ****
-* Selects the task and loads the Google search page of the task at the specified `INDEX`.
+* Selects the task and shows full details of the task at the specified `INDEX`.
 * The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* `1, 2, 3, ...`
+* The index *must be a positive integer* (i.e. `1, 2, 3, ...`).
 ****
 
 Examples:
 
 * `list` +
 `select 2` +
-Selects the 2nd task in the Organizer.
+Selects the 2nd task in the PrioriTask.
 * `find Developer` +
 `select 1` +
 Selects the 1st task in the results of the `find` command.
 
+=== Mark task as complete: `toggle` or `t`
```

Comment:
Would having a command `complete` or `done` be more intuitive to users than `toggle`?